ios/i18n-export.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+KEYMAN_MAC_BASE_PATH="$KEYMAN_ROOT/mac"
+
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+# Please note that this build script (understandably) assumes that it is running on Mac OS X.
+verify_on_mac
+
+## END STANDARD BUILD SCRIPT INCLUDE
+
+BASE_EXPORT_FOLDER="build/localizations"
+START_DIRECTORY=`pwd`
+ARCHIVE_NAME="iOS_localization.zip"
+
+readonly BASE_EXPORT_FOLDER
+readonly START_DIRECTORY
+readonly ARCHIVE_NAME
+
+function exportFile() {
+  BASE_EXPORT_FOLDER="build/localizations"
+
+  if ! [ -d "$BASE_EXPORT_FOLDER" ]; then
+      mkdir -p "$BASE_EXPORT_FOLDER"
+  fi
+
+  LPROJ_SITE=$0
+  LPROJ_BASE=`dirname $0`
+  EXPORT_NAME="en.lproj"
```

Comment:
Does this also need to get exported?
ios\engine\KMEI\KeymanEngine\Classes\LanguagePicker\en.lproj\ResourceInfoView.strings

also affects l. 56

Reply:
Probably so.  Keep in mind, this is very much a draft; most strings still need to be extracted.

Reply:
I found the line. Will move discussion there.